src/structures/ReactionCollector.js

```diff
@@ -81,10 +92,10 @@ class ReactionCollector extends Collector {
    * Handles an incoming reaction for possible collection.
    * @param {MessageReaction} reaction The reaction to possibly collect
    * @param {User} user The user that added the reaction
```

Comment:
This line should be removed (and the method signature needs to be changed in the typings too)

Reply:
I'd argue that it should just stay there: The parameter is still there, since the event emits them both.

Reply:
It's still there for the `collect` event, but not for the `collect` method :eyes: